src/osd/scrubber/pg_scrubber.cc

```diff
@@ -775,7 +775,14 @@ bool PgScrubber::range_intersects_scrub(const hobject_t& start,
 
 Scrub::BlockedRangeWarning PgScrubber::acquire_blocked_alarm()
 {
-  ceph::timespan grace_period{m_debug_blockrange?4s:120s};
+  int grace = get_pg_cct()->_conf->osd_blocked_scrub_grace_period;
+  if (grace == 0) {
+    // we will not be sending any alarms re the blocked object
+    dout(20) << __func__ << ": blocked-alarm disabled" << dendl;
```

Comment:
I think this should say `osd_blocked_scrub_grace_period disabled`. This way, anyone curious about what this configuration is/does can grep for it in the yaml file.

Reply:
@ljflores : fixed (w/ a slightly different message than suggested)